modules/delegates/components/DelegatePicture.tsx

```diff
@@ -158,7 +158,7 @@ export function DelegatePicture({
                   position: 'absolute',
                   bottom: width / -12,
                   right: width / -7,
-                  size: width / 2.5,
+                  width: `${Math.round(width / 2.5)}px`,
```

Comment:
Just wondering why we need this update?

Reply:
I tried modifying the size of the delegate avacatr, and without this, the SVG would go crazy, if you put delegate picture to 20px , then the SVG goes to 500px, with this formula it works better